05_stringer/lexual/stringer_test.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("127.0.0.1\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()")
+		t.Errorf("\nActual: %s\nExpected: %s", actual, expected)
```

Comment:
I'd suggest making this a single string and not calling `t.Errorf` twice for the one failure. While it doesn't look like `testing.T` counts the number of failures, it is possible it might (or might in future). Also since `Errorf` is documented as Log+Fail, calling `Fail` twice for one failure seems a little off.